src/bin/vip-import-sql.js

```diff
@@ -180,13 +212,187 @@ const examples = [
+		await track( 'import_sql_unexpected_tables' );
+		exit.withError( 'The input did not match the expected environment label. Import aborted.' );
+	}
+};
+
+export type validationsAndGetTableNamesInputType = {
+	skipValidate: boolean,
+	appId: number,
+	envId: number,
+	fileNameToUpload: string,
+};
+
+export async function validationsAndGetTableNames( {
+	skipValidate,
+	appId,
+	envId,
+	fileNameToUpload,
+}: validationsAndGetTableNamesInputType ): Promise<Array<string>> {
+	const validations = [ staticSqlValidations, siteTypeValidations ];
+	let tableNamesInSqlFile = [];
+	if ( skipValidate ) {
+		console.log( 'Skipping SQL file validation.' );
+	} else {
+		try {
+			await fileLineValidations( appId, envId, fileNameToUpload, validations );
+		} catch ( validateErr ) {
+			console.log( '' );
+			exit.withError( `${ validateErr.message }
+
+If you are confident the file does not contain unsupported statements, you can retry the command with the ${ chalk.yellow(
+		'--skip-validate'
+	) } option.
+` );
+		}
+		// this can only be called after static validation of the SQL file
+		tableNamesInSqlFile = getTableNames();
+	}
+	return tableNamesInSqlFile;
+}
+
+const getMultiSiteList = async ( { env, track } ): Promise<Array<WPSiteListType>> => {
+	let wpSiteListResults;
+	// get blog_ids from wpcli
+	const api = await API();
+	try {
+		const inputs = {
+			appId: env.appId,
+		};
+		debug( inputs );
+		wpSiteListResults = await api.query( {
+			query: GET_WP_SITE_LIST,
+			variables: inputs,
+		} );
+	} catch ( gqlErr ) {
+		await track( 'import_sql_command_error', {
+			error_type: 'SiteListResults-failed',
+			gql_err: gqlErr,
+		} );
+
+		exit.withError( `wp site list call failed: ${ gqlErr }` );
+	}
+
+	let siteArray = [];
+	if ( Array.isArray( wpSiteListResults?.data?.app?.environments[ 0 ]?.wpSites?.nodes ) ) {
+		siteArray = wpSiteListResults.data?.app.environments[ 0 ].wpSites.nodes;
+	}
+	debug( 'multiSiteArray', siteArray );
+	return siteArray;
+};
+
+const displayPlaybook = async ( {
+	launched,
+	tableNames,
+	searchReplace,
+	fileName,
+	domain,
+	formattedEnvironment,
+	isMultiSite,
+	app,
+	env,
+	track,
+} ) => {
+	console.log();
+	console.log( `  importing: ${ chalk.blueBright( fileName ) }` );
+	console.log( `         to: ${ chalk.cyan( domain ) }` );
+	console.log( `       site: ${ app.name } (${ formattedEnvironment })` );
+
+	if ( searchReplace?.length ) {
+		const output = ( from, to ) => {
+			const message = `        s-r: ${ chalk.blue( from ) } -> ${ chalk.blue( to ) }`;
+			console.log( message );
+		};
+
+		formatSearchReplaceValues( searchReplace, output );
+	}
+
+	let siteArray = [];
+	if ( isMultiSite ) {
+		// eslint-disable-next-line no-multi-spaces
+		console.log( `  multisite: ${ isMultiSite.toString() }` );
+		siteArray = await getMultiSiteList( { env, track } );
+	}
+
+	if ( ! tableNames.length ) {
+		console.log();
+		console.log( 'Since validation was skipped, no playbook information could be displayed' );
```

Comment:
I'd probably just not print anything in this case.  Maybe change these to `debug`s?